ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.spec.ts

```diff
@@ -0,0 +1,366 @@
+import { async, ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { AddressEmbeddedComponent } from './addressEmbedded.component';
+import { AddressInputComponent } from './addressInput.component';
+import { ValidationMessage } from '../validationMessage.component';
+import { MatCardModule, MatRadioButton, MatRadioGroup, MatRadioModule } from '@angular/material';
+import { ReactiveFormsModule } from '@angular/forms';
+import { Component, DebugElement, EventEmitter, Input, Output } from '@angular/core';
+
+import { of, Subject, throwError } from 'rxjs';
+import { By } from '@angular/platform-browser';
+import { AddressVerificationStatus } from '../../models/addressVerificationStatus';
+import { cold, getTestScheduler, hot } from 'jasmine-marbles';
+import { ActivityResponse, Address, AddressService } from 'ddp-sdk';
```

Comment:
We shouldn't import it from SDK

Reply:
Fixed in 5e382169ece0a482f548ae9f7e9b61153f20a165